pages/proposals/proposals.jsx

```diff
@@ -158,25 +193,24 @@ var Proposal = React.createClass({
             type="submit"
             onClick={this.handleFormSubmit}
             disabled={this.state.submitting ? `disabled` : null}
-          >{ this.state.submitting ? SUBMITTING_LABEL : PRE_SUBMIT_LABEL }</button>
+          >{ this.state.submitting ? `Submiting...` : stringSource.form_field_controls.submit }</button>
           { this.state.showFormInvalidNotice && <div className="d-inline-block form-invalid-error">Something isn't right. Please fix the errors indicated above.</div> }
         </div>
 
         { !this.state.submitting && this.state.submissionStatus === SUBMISSION_STATUS_FAIL && this.renderSubmissionFail() }
       </div>
     )
   },
-  renderIntro() {
+  renderIntro(content) {
+    let paragraphs = content.body.map((paragraph, i) => {
+      // using dangerouslySetInnerHTML is okay here since we are pulling strings from a static json file, not user content or anything change dynamically over time without our attention
+      return <p dangerouslySetInnerHTML={{__html: paragraph}} key={i}></p>;
```

Comment:
no. Unless you bundle them in, because then you could makebabel+webpack interpret the code as JSX (I remember we did this in learning.moz a few times?)